src/mudlet.cpp

```diff
@@ -714,246 +418,611 @@ mudlet::mudlet()
-    optionsKeySequence = QKeySequence(Qt::CTRL | Qt::Key_P);
-    notepadKeySequence = QKeySequence(Qt::CTRL | Qt::Key_N);
-    packagesKeySequence = QKeySequence(Qt::CTRL | Qt::Key_O);
-    modulesKeySequence = QKeySequence(Qt::CTRL | Qt::Key_I);
-    multiViewKeySequence = QKeySequence(Qt::CTRL | Qt::ALT | Qt::Key_V);
-    connectKeySequence = QKeySequence(Qt::CTRL | Qt::ALT | Qt::Key_C);
-    disconnectKeySequence = QKeySequence(Qt::CTRL | Qt::Key_D);
-    reconnectKeySequence = QKeySequence(Qt::CTRL | Qt::Key_R);
-#else
-    triggersKeySequence = QKeySequence(Qt::ALT | Qt::Key_E);
-    showMapKeySequence = QKeySequence(Qt::ALT | Qt::Key_M);
-    inputLineKeySequence = QKeySequence(Qt::ALT | Qt::Key_L);
-    optionsKeySequence = QKeySequence(Qt::ALT | Qt::Key_P);
-    notepadKeySequence = QKeySequence(Qt::ALT | Qt::Key_N);
-    packagesKeySequence = QKeySequence(Qt::ALT | Qt::Key_O);
-    modulesKeySequence = QKeySequence(Qt::ALT | Qt::Key_I);
-    multiViewKeySequence = QKeySequence(Qt::ALT | Qt::Key_V);
-    connectKeySequence = QKeySequence(Qt::ALT | Qt::Key_C);
-    disconnectKeySequence = QKeySequence(Qt::ALT | Qt::Key_D);
-    reconnectKeySequence = QKeySequence(Qt::ALT | Qt::Key_R);
-#endif
-    connect(this, &mudlet::signal_menuBarVisibilityChanged, this, &mudlet::slot_update_shortcuts);
 
-    mpSettings = getQSettings();
-    readLateSettings(*mpSettings);
-    // The previous line will set an option used in the slot method:
-    connect(mpMainToolBar, &QToolBar::visibilityChanged, this, &mudlet::slot_handleToolbarVisibilityChanged);
-
-#if defined(INCLUDE_UPDATER)
-    updater = new Updater(this, mpSettings);
-    connect(updater, &Updater::signal_updateAvailable, this, &mudlet::slot_updateAvailable);
-    connect(dactionUpdate, &QAction::triggered, this, &mudlet::slot_check_manual_update);
-#if defined(Q_OS_MACOS)
-    // ensure that 'Check for updates' is under the Applications menu per convention
-    dactionUpdate->setMenuRole(QAction::ApplicationSpecificRole);
-#else
-    connect(updater, &Updater::signal_updateInstalled, this, &mudlet::slot_update_installed);
-#endif // !Q_OS_MACOS
-#endif // INCLUDE_UPDATER
-
-    if (!mToolbarIconSize) {
-        setToolBarIconSize(mEnableFullScreenMode ? 2 : 3);
+    for (auto& translationFileName : qAsConst(translationFilesList)) {
+        QString languageCode(translationFileName);
+        languageCode.remove(QStringLiteral("mudlet_"), Qt::CaseInsensitive);
+        languageCode.remove(QStringLiteral(".qm"), Qt::CaseInsensitive);
+        int percentageTranslated = -1;
+
+        auto* pTranslator = new QTranslator();
+        std::unique_ptr<QTranslator> pMudletTranslator(pTranslator);
+        auto translationPathFileName(path % QLatin1Char('/') % translationFileName);
+        if (Q_LIKELY(pMudletTranslator->load(translationFileName, path))) {
+            qDebug().noquote().nospace() << "    found a Mudlet translation for locale code: \"" << languageCode << "\".";
```

Comment:
I thought **release** versions have `qDebug()` and related messages nulled out.  OTOH The list of translations listed here are what drives finding the corresponding Qt library ones and those can be missing {like I found on my FreeBSD setup recently} so having both sets of them listed might be useful if they do get shown for the end users just in case something is amiss in their system for the *Qt* ones...

tl;dr; For the moment yes - if all goes well we can probably afford to lose them in a later 4.x release...